src/provable/binable.ts

```diff
@@ -175,3 +116,22 @@ function bytesToBits(bytes: number[]) {
     })
     .flat();
 }
+
```

Comment:
This could use some comments describing what it does, preconditions and postconditions